ktor-server/ktor-server-plugins/ktor-server-metrics-micrometer/jvm/src/io/ktor/server/metrics/micrometer/MicrometerMetrics.kt

```diff
@@ -119,6 +119,34 @@ public class MicrometerMetricsConfig {
     public fun timers(block: Timer.Builder.(ApplicationCall, Throwable?) -> Unit) {
         timerBuilder = block
     }
+
+    internal var transformRoute: RoutingNode.() -> String = { path }
```

Comment:
Note: We should mark this issue as "impactful change" as it requires migration for those who relies on the old behavior